app/controllers/triggers_controller.rb

```diff
@@ -1,5 +1,6 @@
 class TriggersController < ApplicationController
   before_action :set_trigger, only: [:show, :edit, :update, :destroy]
+  before_action :authenticate_user!, only: [:new, :edit, :create, :update, :destroy] unless Rails.env.test?
```

Comment:
You have to change the respective tests. Add `Given I am logged in` where necessary.